docs/reference/receiver.md

```diff
@@ -541,8 +541,8 @@ It is illegal to call this function if the receiver's queue is empty (i.e. when
 The `wb_receiver_get_emitter_direction` function also operates on the head packet in the receiver's queue.
 It returns a normalized (length=1) vector that indicates the direction of the emitter with respect to the receiver's coordinate system.
 The three vector components indicate the *x, y *, and *z*-directions of the emitter, respectively.
-For example, if the emitter was exactly in front of the receiver, then the vector would be [0 0 1].
-In the usual orientation used for 2D simulations (robots moving in the *xz*-plane and the *y *-axis oriented upwards), a positive *x *-component indicates that the emitter is located to the left of the receiver while a negative *x *-component indicates that the emitter is located to the right.
+For example, if the emitter was exactly in front of the receiver, then the vector would be [1 0 0].
+In the usual orientation used for 2D simulations (robots moving in the *xy*-plane and the *z *-axis oriented upwards), a positive *y *-component indicates that the emitter is located to the left of the receiver while a negative *y *-component indicates that the emitter is located to the right.
```

Comment:
Note that this sentence is not rendered correctly (`*z *-axis`, etc) and the same issue occurs with line 543:
https://www.cyberbotics.com/doc/reference/receiver?version=feature-flu-emitter-receiver

This error was already there in the previous version, but it would be better to fix it.